packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -239,6 +240,25 @@ export class DashboardService {
         const duplicatedDashboard = {
             ...dashboard,
             name: `Copy of ${dashboard.name}`,
+            tiles: dashboard.tiles.map<CreateDashboard['tiles'][number]>(
+                (tile) => {
+                    switch (tile.type) {
+                        case DashboardTileTypes.SAVED_CHART: {
+                            const { properties, ...rest } = tile;
+                            return <CreateDashboardChartTile>{
+                                ...rest,
+                                properties: {
+                                    title: properties.title,
+                                    hideTitle: properties.hideTitle,
+                                    savedChartUuid: properties.savedChartUuid,
```

Comment:
struggling to understand why we wouldn't want this chart to have `belongsToDashboard` set to `true` 🤔

Reply:
Because `belongsToDashboard` shouldn't be part of the creation objects. I had to rollback that typing restriction because of https://github.com/lightdash/lightdash/issues/6450
I think I'm going to revert all these changes related to the type restriction as it is confusing now.

Reply:
ah, I see! Sure 👍